packages/orchestration/src/cosmos-api.ts

```diff
@@ -101,14 +102,18 @@ export interface CosmosAssetInfo extends Record<string, unknown> {
 /**
  * Info for a Cosmos-based chain.
  */
-export type CosmosChainInfo = Readonly<{
+export interface CosmosChainInfo extends BaseChainInfo {
   /** can be used to lookup chainInfo (chainId) from an address value */
   bech32Prefix?: string;
+  /** plain reference (gaia-1) or CAIP-2 (cosmos:gaia-1) supported */
   chainId: string;
-
-  connections?: Record<string, IBCConnectionInfo>; // chainId or wellKnownName
+  /**  IBC connections between this chain and others, keyed by chainId */
+  connections?: Record<string, IBCConnectionInfo>;
+  /** indicates the host chain supports Interchain Accounts (ICS-27) */
+  icaEnabled?: boolean;
   // UNTIL https://github.com/Agoric/agoric-sdk/issues/9326
```

Comment:
```suggestion
  /** indicates the host chain supports Interchain Accounts (ICS-27). Not currently used at runtime - only for types */
  icaEnabled?: boolean;
  /**
   * indicates the host chain support the async-icq IBC Application protocol. Used at runtime to permit `Chain.query()` operations.
   *
   * in the future, querying will be supported by the ICA protocol natively and this will likely deprecate. @see https://github.com/Agoric/agoric-sdk/issues/9326
   */
```